src/app/routes/utils/regex/utils/index.js

```diff
@@ -35,11 +30,17 @@ const getWorldServices = services => {
     'cymrufyw',
     'naidheachdan',
     'archive',
+    'scotland',
   ];
 
   return services.filter(service => !publicServices.includes(service));
 };
 
+export const getArticleSwRegex = services => {
+  const serviceRegex = getServiceRegex(getWorldServices(services));
+  return `/:service(${serviceRegex})/:local(${articleLocalRegex})/sw.js`;
+};
+
 export const getArticleManifestRegex = services => {
   const serviceRegex = getServiceRegex(getWorldServices(services));
   return `/:service(${serviceRegex})/:local(${articleLocalRegex})/manifest.json`;
```

Comment:
~This is potentially out of scope (or is it? - Let's get a second opinion!)~ Let's do this while we're here: per my comment above anything that uses `getWorldServices` no longer needs `articleLocalRegex`. If we did change it, we would also need to update some tests.

(Note that this we recently removed manifest paths from Public Services, and we just missed this.

```suggestion
export const getArticleManifestRegex = services => {
  const serviceRegex = getServiceRegex(getWorldServices(services));
  return `/:service(${serviceRegex})/articles/manifest.json`;
```

Reply:
That makes sense. Updated